mantidproject/mantid
# Allow PreviewPlot to toggle errors per curve

Fixes #12877.

To test:
- Open Indirect > Data Analysis > MSDFit
- Open `Babylon5/Public/DanNixon/data/osi92762_graphite002_elwin_eq2.nxs`
- See that Sample curve has errors
- Run
- See that Fit does not have errors
- Toggle error bars via context menu
- See that when errors are added to Fit, they are still there after the algorithm is run again

No release notes updates as this is covered in the mention of last changes to PreviewPlot.
